README.md

```diff
@@ -336,9 +336,51 @@ bin/console assets:install
     If you are using Sylius version >= 1.12 then Node version 18 is fully supported.
 
 1. Install dependencies:
-    
+
     for Sylius <= 1.11:
     ```bash
+    composer require symfony/webpack-encore-bundle
```

Comment:
One thing, couldn't we just add it to composer?
On <=1.11 it should install the fitting version based on other constraints, while >.11 it'll take the version set on Sylius.